### ISSUE TITLE
Dependency conflicts when converting to maven

### ISSUE THREAD

**Issue by ameenhere** [simpledynamics/openid-connect-server-spring-boot#5]
This is a question basically. I tried to convert the project to use maven instead of gradle. But faced several dependency conflicts. spring-beans version was overridden to 4.0.9 due to spring-security-oauth dependency and so on. The compiler started to complain of various things. Any guidance on how can I transition your project to use maven? What I basically tried was to use the pom file generated in the build folder and use it as the main pom.xml file.

**Comment by barretttucker** [comment]
Do you want to build the project using maven or just use the project as a dependency in a maven project?

**Comment by ameenhere** [comment]
I wanted to build the project with maven and customize the project to suit our needs, I have forked your project for the same. I have resolved the conflicts manually and its working now. I installed it using gradle ,used the pom file uploaded in the local maven repository, and tweaked it to exclude certain conflicting dependencies.  I am new to gradle and couldnt grasp how gradle got the dependency conflicts resolved correctly whereas maven could not. I think this question can be closed, as there is no straightforward way to transition into a maven project from gradle.
